bot.py

```diff
@@ -186,7 +189,7 @@ def __fetch_feed(self, feed_info):
                             newsdate = "no date"
 
                     # Update the database. If it's a new issue, post it to the channel
-                    is_new = self.__db.insert_news(feed_info[0], newstitle, newsurl, newsdate)
+                    is_new = self.__db.insert_news(feed_info[0], newstitle, newsitem.link, newsdate)
```

Comment:
Why did you change this? Wouldn't this make the whole `newsurls` stuff in line 177 unnecessary, when the raw `newsitem.link` value is used? I guess this should stay `newsurl`?